fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -2902,11 +2902,11 @@ ACTOR void simulationSetupAndRun(std::string dataFolder,
 		testConfig.storageEngineExcludeTypes.insert(SimulationStorageEngine::SHARDED_ROCKSDB);
```

Comment:
Could we disable other storage engines when SHARD_ENCODE_LOCATION_METADATA is enabled like 7.3 branch?

Reply:
I feel at least, we should enable SQLite, RocksDB, ShardedRocksDB for this case. I will disable other engines.